src/super_gradients/training/models/detection_models/csp_darknet53.py

```diff
@@ -215,18 +233,23 @@ def __init__(self, arch_params: HpmStruct):
         else:
             raise NotImplementedError(f'One of {yolo_type} yolo type or {yolo_version} yolo version is not supported')
 
-        self._modules_list.append(Conv(width_mult(64), width_mult(128), 3, 2, activation_type))                      # 1
-        self._modules_list.append(block(width_mult(128), width_mult(128), struct[0], activation_type))               # 2
-        self._modules_list.append(Conv(width_mult(128), width_mult(256), 3, 2, activation_type))                     # 3
-        self._modules_list.append(block(width_mult(256), width_mult(256), struct[1], activation_type))               # 4
-        self._modules_list.append(Conv(width_mult(256), width_mult(512), 3, 2, activation_type))                     # 5
-        self._modules_list.append(block(width_mult(512), width_mult(512), struct[2], activation_type))               # 6
-        self._modules_list.append(Conv(width_mult(512), width_mult(1024), 3, 2, activation_type))                    # 7
+        self._modules_list.append(ConvBlock(width_mult(64), width_mult(128), 3, 2, activation_type))                 # 1
```

Comment:
this part of the code really looks like it should have been a for loop (lines 236 to 245)

Reply:
Added in [license and validity method added to yolox loss](https://github.com/Deci-AI/super-gradients/pull/210/commits/13b40aea02f859cfac67d72ea4967b5ec761d0d0)

Reply:
done.